modules/applications/pages/ui-builder/how-to-configure-https.adoc

```diff
@@ -0,0 +1,141 @@
+= HTTPS Configuration
+:page-aliases: applications:how-to-configure-https.adoc
+:description: In production, it is necessary to set up both Bonita Runtime and UI Builder for HTTPS instead of HTTP. The easiest way it to configure Bonita UI Proxy for SSL offloading.
```

Comment:
```suggestion
:description: In production, it is necessary to set up both Bonita Runtime and UI Builder for HTTPS instead of HTTP. The easiest way is to configure Bonita UI Proxy for SSL offloading.
```